src/tike/ptycho/solvers/divided.py

```diff
@@ -127,13 +127,13 @@ def lstsq_grad(
     return result
 
 
-def _get_nearplane_gradients(nearplane, psi, scan_, probe, unique_probe,
-                             op, m, recover_psi, recover_probe):
+def _get_nearplane_gradients(nearplane, psi, scan_, probe, unique_probe, op, m,
+                             recover_psi, recover_probe):
 
     pad, end = op.diffraction.pad, op.diffraction.end
 
     patches = op.diffraction.patch.fwd(
-        patches=cp.zeros(nearplane[..., m:m + 1, pad:end, pad:end].shape,
```

Comment:
And why change to only m?

Reply:
`m:m + 1` is equivalent to `[m]` in this context. I thought it might look clearer. Is it less clear?